pom.xml

```diff
@@ -63,7 +63,8 @@
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.479</jenkins.version>
+        <!-- TODO: Waiting for JENKINS-73824 and JENKINS-73835 to make it into an LTS line -->
+        <jenkins.version>2.481</jenkins.version>
```

Comment:
Up to reviewers what we want to do here. The removal of the call to `logRotate` in `WorkflowRun.finish` eliminates a redundancy, but things should be fine for 2.481+ users even with it in place. We can either merge this as-is or wait for the fixes to possibly get backported.